app/create/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var Angular    = require('angular');
+var Controller = require('./controller.js');
```

Comment:
Also, we usually separate the global requires like angular from the local requires like the controller and state with a new line